functions/test/firebase/firestore_test.rb

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "helper"
+
+describe "functions_firebase_firestore" do
+  include FunctionsFramework::Testing
+
+  let(:source) { "my-resource" }
+  let(:type) { "type" }
```

Comment:
I copied the event type from the [Node test](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/firebase/test/index.test.js#L95). I'm not sure what a realistic value would be.